### PR TITLE
Add session autorenew

### PR DESCRIPTION
**Pull request by dnfd**

### CODE REVIEW

File: config/initializers/barong.rb
Comment (ec) [inline config/initializers/barong.rb]: value should be in strings
lets stick to 'enabled' and 'disabled'

also var name should be prefixed with 'barong'

**Review by ec — CHANGES_REQUESTED** [review]

**Comment by ec** [comment]
This patch is conflicting with validations patch. Lets wait until we merge #904 first

**Review by ec — APPROVED** [review]

**Review by ec — APPROVED** [review]

**Review by mod — CHANGES_REQUESTED** [review]
I don't agree with having a param
To me this should work well out of the box
What's the usecase ?
What are the API calls which triggers renew

More important is security like redis session and CSRF token on login

**Pull request merged**
